src/SingleBookingPage/bookingItem/Accordion.js

```diff
@@ -35,7 +35,9 @@ class Accordion extends React.Component<Props, State> {
     isToggled: false,
   };
   toggleBody = () => {
-    this.setState({ isToggled: !this.state.isToggled });
+    this.setState(prevState => {
```

Comment:
you need to return an object with the new state, like this, it will think it is just a function, wrap it with `()`
`this.setState(prevState => ({isToggled: !prevState.isToggled}))` or 
`this.setState(prevState => { 
      return {isToggled: !prevState.isToggled}
})`